apps/site/lib/site_web/views/schedule/timetable.ex

```diff
@@ -101,4 +120,12 @@ defmodule SiteWeb.ScheduleView.Timetable do
   @spec cell_via_class(String.t() | nil) :: String.t()
   def cell_via_class(nil), do: ""
   def cell_via_class(<<_::binary>>), do: " m-timetable__cell--via"
+
+  @spec trains_or_boats(integer) :: String.t()
+  def trains_or_boats(2), do: "train"
+  def trains_or_boats(4), do: "boat"
+
+  @spec is_ferry(integer) :: boolean
+  def is_ferry(4), do: true
+  def is_ferry(_), do: false
```

Comment:
We often identify route modes by their `type_atom`, so I think this would be better defined as `def is_ferry(route), do: Routes.Route.type_atom(route) == :ferry`